File: Week-02/Day-1/seconds_in_a_day.py
current_hours = 14
current_minutes = 34
current_seconds = 42

# Write a program that prints the remaining seconds (as an integer) from a
# day if the current time is represented bt the variables
day = int(60*60*24)
print (day)
current_time_in_sec = int(current_hours)*60*60 + int(current_minutes)*60 + int(current_seconds)
print (current_time_in_sec)
remaining_time = day - current_time_in_sec
print (remaining_time)

File: Week-07/Day-1/seconds-in-a-day.js

'use strict';

var currentHours = 14;
var currentMinutes = 34;
var currentSeconds = 42;

// Write a program that prints the remaining seconds (as an integer) from a
// day if the current time is represented by these variables


let oneDay = 60*60*24;
let remaininseconds = oneDay - currentHours*60*60 - currentMinutes*60 - currentSeconds
console.log(remaininseconds)